library/src/window_impl.cxx

```diff
@@ -44,6 +55,78 @@ class window_impl::internals
     return this->CachePath;
   }
 
+#if _WIN32
+  /**
+   * Helper function to detect if the
+   * Windows Build Number is equal or greater to a number
+   */
+  bool IsWindowsBuildNumberOrGreater(int buildNumber)
+  {
+    std::string value{};
+    bool result = vtksys::SystemTools::ReadRegistryValue(
+      "HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Windows NT\\CurrentVersion;CurrentBuildNumber",
+      value);
+
+    if (result == true)
+    {
+      try
+      {
+        return std::stoi(value) >= buildNumber;
+      }
+      catch (const std::invalid_argument& e)
+      {
+        f3d::log::debug("Error parsing CurrentBuildNumber", e.what());
```

Comment:
Should be an error message, not a debug

Reply:
On older versions of windows this will likely fail and is just ignored. So, I moved it to debug. Let me know what you think.

Reply:
Ok then I'm ok with the debug level.